Pod/Classes/UI/NumberInputTextField.swift

```diff
@@ -171,9 +171,10 @@ public class NumberInputTextField: StylizedTextField {
         let numberPartiallyValid = type.checkCardNumberPartiallyValid(cardNumber) == .Valid
         
         if numberPartiallyValid {
-            let formatter = cardNumberFormatter
-            text = formatter.formattedCardNumber(unformattedCardNumber)
-            numberInputTextFieldDelegate?.numberInputTextFieldDidChangeText(self)
```

Comment:
I assume that calling this delegate was causing the issue. Are there any side effects by not calling it anymore?